frontoffice/src/resources/Activity/Event/EventList.js

```diff
@@ -52,7 +52,7 @@ const EventList = (props) => {
           icon: ListIcon,
           perPage: 1000,
           sort: { field: 'pair:startDate', order: 'ASC' },
-          filterDefaultValues: { sparqlWhere: futureEventSparql, dereference: [] },
+          filterDefaultValues: { sparqlWhere: futureEventSparql },
```

Comment:
Hmm pourquoi tu enlèves `dereference: []` ? Ca permet d'éviter qu'on fetch la localisation de l'événement (`pair:hasLocation`) lorsqu'on l'affiche sous forme de liste. Ca améliore pas mal les performances.

Reply:
Ok, c'est parce que ça me générait des erreurs (Cannot read properties of undefined (reading 'url')) et que ça ne m'affiche aucun événement quand je charge la page mais ça doit être seulement en local

Reply:
C'est sans doute parce que tu dois merger master dans cette branche. `dereference` s'appelle maintenant `blankNodes`.